src/jupyter_contrib_nbextensions/nbextensions/accessibility_toolbar/font_style/font_control.js

```diff
@@ -0,0 +1,173 @@
+        $("<option>", {
+          value: "'Palatino Linotype', 'Book Antiqua', Palatino, serif"
+        }).text("Palatino Linotype")
+      );
+      fs_font_name.append(
+        $("<option>", { value: "Tahoma, Geneva, sans-serif" }).text("Tahoma")
+      );
+      fs_font_name.append(
+        $("<option>", { value: "'Times New Roman', Times, serif" }).text(
+          "Times New Roman"
+        )
+      );
+      fs_font_name.append(
+        $("<option>", { value: "Verdana, Geneva, sans-serif" }).text("Verdana")
+      );
+      return fs_font_name;
+    };
+
+    Font_control.prototype.font_size = function() {
+      var fs_font_size = $("<select>", {
+        id: "font_size",
+        class: "select-box"
+      });
+      fs_font_size.append($("<option>", { value: "10" }).text("10"));
+      fs_font_size.append($("<option>", { value: "12" }).text("12"));
+      fs_font_size.append(
+        $("<option>", { value: "14", selected: "selected" }).text("14")
+      );
+      fs_font_size.append($("<option>", { value: "20" }).text("20"));
+      fs_font_size.append($("<option>", { value: "28" }).text("28"));
+      fs_font_size.append($("<option>", { value: "32" }).text("32"));
+      fs_font_size.append($("<option>", { value: "48" }).text("48"));
+      fs_font_size.append($("<option>", { value: "56" }).text("56"));
+      fs_font_size.append($("<option>", { value: "64" }).text("64"));
+      fs_font_size.append($("<option>", { value: "72" }).text("72"));
+      return fs_font_size;
+    };
+
+    function set_font_name(name) {
+      for (
+        var index = 0;
+        index < document.getElementsByClassName("CodeMirror").length;
+        index++
+      ) {
+        document.getElementsByClassName("CodeMirror")[
+          index
+        ].style.fontFamily = name;
+        document.getElementsByClassName("inner_cell")[
+          index
+        ].style.fontFamily = name;
+        if (Jupyter.notebook.get_cell(index).cell_type == "code") {
+          document.getElementsByClassName("output_subarea")[
+            index
+          ].children[0].style.fontFamily = name;
+        }
+      }
+    }
+
+    function set_font_size(size) {
+      for (
+        var index = 0;
+        index < document.getElementsByClassName("CodeMirror").length;
+        index++
+      ) {
+        document.getElementsByClassName("CodeMirror")[index].style.fontSize =
+          size + "px";
+        document.getElementsByClassName("inner_cell")[index].style.fontSize =
+          size + "px";
+        if (Jupyter.notebook.get_cell(index).cell_type == "code") {
+          document.getElementsByClassName("output_subarea")[
+            index
+          ].style.fontSize = size + "px";
+        }
+      }
+    }
+
+    Font_control.prototype.get_font_name = function() {
+      return this.fontName;
+    };
+
+    Font_control.prototype.get_font_size = function() {
+      return this.fontSize;
+    };
+
+    Font_control.prototype.font_control = function() {
```

Comment:
Unless this is a constructor we shouldn't make it the same name as the object/class